### PR TITLE
Add exception handling for unexpected errors from google-cloud-logging.

### PR DESCRIPTION
**Pull request by qingling128**
Workaround for https://github.com/GoogleCloudPlatform/google-cloud-ruby/issues/1893

### CODE REVIEW

**Comment by qingling128** [comment]
Friendly ping.

File: lib/fluent/plugin/out_google_cloud.rb [hunk 1]
```diff
@@ -656,6 +656,15 @@ def write(chunk)
...
                          error: error.to_s, error_code: error.code.to_s
             end
+
+          # Got an unexpected error (not Google::Gax::GaxError) from the
+          # google-cloud-logging lib.
+          rescue StandardError => error
+            increment_failed_requests_count(error.code)
+            increment_dropped_entries_count(entries_count, error.code)
+            @log.error "Unexpected error type #{error.class.name} from the "\
+                       "client lib, dropping #{entries_count} log message(s)",
```
Comment (igorpeshansky) [inline lib/fluent/plugin/out_google_cloud.rb]: s/`lib`/`library`/?

File: lib/fluent/plugin/out_google_cloud.rb [hunk 2]
```diff
@@ -656,6 +656,15 @@ def write(chunk)
                          "server, dropping #{entries_count} log message(s)",
                          error: error.to_s, error_code: error.code.to_s
             end
+
+          # Got an unexpected error (not Google::Gax::GaxError) from the
+          # google-cloud-logging lib.
+          rescue StandardError => error
+            increment_failed_requests_count(error.code)
```
Comment (igorpeshansky) [inline lib/fluent/plugin/out_google_cloud.rb]: A `StandardError` won't have a `code` field or method, would it?

Comment (qingling128) [reply lib/fluent/plugin/out_google_cloud.rb]: Changed it to `GRPC::Core::StatusCodes::UNKNOWN` instead.

File: lib/fluent/plugin/out_google_cloud.rb (hunk 1 above)
Comment (qingling128) [reply lib/fluent/plugin/out_google_cloud.rb]: Changed.

**Comment by qingling128** [comment]
Gentle reminder.

**Pull request merged**